src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Aggregator/AggregatorConfiguration.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Configuration options for function result aggregation.
+    /// </summary>
+    public class AggregatorConfiguration
```

Comment:
Done. Named it `FunctionResultAggregatorConfiguration` to match with other classes.